process-wpt-results.js

```diff
@@ -205,28 +185,24 @@ export function get_focus_areas () {
         area_names[key] = area.name
     }
 
-    area_keys.sort((a, b) => FOCUS_AREAS[a].order - FOCUS_AREAS[b].order)
     return { area_keys, area_names }
```

Comment:
Has this change been tested with the frontend, by generating a new scores.json? The `area_keys` is used by the [front end to construct the drop down items](https://github.com/servo/servo.org/blob/a97ae951556eee8091d32ab34ddee775afe7befc/assets/js/load-chart.js#L167) for the area selection input.

Reply:
I tested manually. The json was like

```js
{
  "area_keys": ["all", "csp", "css", ...],
  "focus_areas": {
    "all": "All WPT tests",
    "csp": "/content-security-policy",
    "css": "/css",
  },
  "scores": [...]
}
```
and now becomes

```js
{
  "area_keys": ["0", "1", "2", ...],
  "focus_areas": {
    "0": "All WPT tests",
    "1": "/content-security-policy",
    "2": "/css",
    ...
  },
  "scores": [...]
}
```
so the front-end shouldn't have any problem.

A follow-up should just remove `area_keys` and turn `focus_areas` into an array, but that will require frontend changes.